robot_init_state_param:
- -0.0011162976781255952
- -0.00022108915677918975
- 0.8380643327658639
- 0.9996573184912552
- 4.209265470576367e-07
- 0.026177195928605696
- 1.8946783540182179e-07
- 0.01877976675484756
- 0.000982887444102318
- -0.4578332727977961
- 0.841593685076642
- -0.4361123500950203
- -0.018807672090764276
- -0.01815062200418652
- -0.0009518097051129123
- -0.45786657188903307
- 0.8416536674493815
- -0.43614019538078613
- 0.018174760962873054
